build/images/scripts/install_cni_chaining

```diff
@@ -12,6 +12,11 @@ while true; do
   sleep 2s
 done
 cni_conf="/host/etc/cni/net.d/$cni_conf"
+
+if grep -sq "azure" $cni_conf; then
+  sed -i 's/"mode":"bridge",/"mode":"transparent",/g' $cni_conf
+fi
```

Comment:
should we have a separate `install_cni_chaining_aks` file to avoid confusion in other clouds? I don't have a string preference...

Reply:
I think azure config check to detect cloud is good enough. Keeping single file is easy for readable. If you are concern about string check, we can also detect cloud from meta data service. Probably another PR.